package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "koa-gu",
-  "version": "0.0.16",
+  "name": "@guardian/koa-gu",
+  "version": "0.0.2",
```

Comment:
This is a new package (as https://www.npmjs.com/package/@guardian/koa-gu  404s). Should we start at 0.0.1?

Reply:
I wanted to give the sense that it isn't box-fresh code; maybe the better thing would be to number it at 0.0.1 and include in the description that it is a descendant of the other library